apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/cql/ExpressionUtils.java

```diff
@@ -0,0 +1,33 @@
+package io.stargate.sgv2.api.common.cql;
+
+import com.bpodgursky.jbool_expressions.And;
+import com.bpodgursky.jbool_expressions.Expression;
+import com.bpodgursky.jbool_expressions.Or;
+import java.util.List;
+
+public class ExpressionUtils<K> {
```

Comment:
Add a simple javadoc explaining its set of utility methods for constructing logical expressions for use by QueryBuilder.

Reply:
done

Reply:
Hmmh. I don't see any for the class itself? (Javadoc comment as in /** .... */ )

Reply:
Ok good!